### ISSUE TITLE
PLL numbers swapped for Bergen Board

### ISSUE THREAD

**Issue by KyleM32767** [newaetech/cw310-bergen-board#10]
On the CW310 Bergen Board, the numbers of the PLL outputs when calling `target.pll` methods do not correspond to the correct PLL output numbers. (i.e. calling `target.pll.pll_outenable_set(True, 2)` enables PLL output 1, and vice versa).

Looking at the schematic and constraint file, it seems like this is because the signals for PLL_CLK1 and PLL_CLK2 are on pins Y4 and Y1, respectively of the external PLL. 

![image](https://github.com/user-attachments/assets/0402a358-f084-4a12-8450-973c39b94136)

```
#####
# Input Clocks
#####

set_property -dict { PACKAGE_PIN  R22  IOSTANDARD   LVCMOS33 } [get_ports { PLL_CLK1 }]; #IO_L14P_T2_SRCC_13
set_property -dict { PACKAGE_PIN  N21  IOSTANDARD   LVCMOS33 } [get_ports { PLL_CLK2 }]; #IO_L12P_T1_MRCC_13
```

However [the API](https://github.com/newaetech/chipwhisperer/blob/develop/software/chipwhisperer/hardware/naeusb/pll_cdce906.py) assumes PLL_CLK1 is on pin Y1 and PLL_CLK2 is on Y4...

```py
    def outputUpdateOutputs(self, outnum, pllsrc_new=None, pllenabled_new=None, pllslewrate_new=None):
        """Update the output pins with enabled/disabled, slew rate, etc"""
        outnum = self.swap_340_pllnum(outnum)
        # Map to output pins on CDCE906 Chip
        if outnum == 0:
            outpin = 0
            if pllsrc_new is None:
                src = self._pll0source
            else:
                src = pllsrc_new

            if src == 'PLL0':
                divsrc = 0
            elif src == 'PLL1':
                divsrc = 1
            elif src == 'PLL2':
                divsrc = 2
        elif outnum == 1:
            outpin = 1
            divsrc = 1
        elif outnum == 2:
            outpin = 4
            divsrc = 2
```

The CW305 is that way, which tells me this is just a mistake with the CW310 design.

![image](https://github.com/user-attachments/assets/6222f9f1-8221-410d-b1b2-f02fd45e34ca)

**TLDR: Switch the labels for PLL_CLK1 and PLL_CLK2 on the schematic and `bergen.xdc`**
